### PR TITLE
Remove timeout for messages and disable grub timeout in AY profiles

### PR DESCRIPTION
**Pull request by rwx788**
See [poo#37015](https://progress.opensuse.org/issues/37015).

[Verification run](http://g226.suse.de/tests/2031).

### CODE REVIEW

File: data/autoyast/autoyast_btrfs.xml [hunk 1]
```diff
@@ -166,4 +171,26 @@
       <username>root</username>
     </user>
   </users>
+  <report>
+    <errors>
+      <log config:type="boolean">true</log>
```
Comment (SergioAtSUSE) [inline data/autoyast/autoyast_btrfs.xml]: As spoken, we need to find out if <log...> and <show...> are really needed here

Comment (rwx788) [reply data/autoyast/autoyast_btrfs.xml]: These are indeed default values, so removed from given profiles. Thanks!

**Comment by SergioAtSUSE** [comment]
Travis complaints

**Comment by rwx788** [comment]
Travis is now fixed, re-triggered it.

**Pull request merged**